### PR TITLE
Multiple server details get from json data added

### PR DESCRIPTION
**Pull request by vaishaliagola27**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -63,6 +63,14 @@ jobs:
...
           name: SSH Key setup
           command: echo -e "Host *\n\tStrictHostKeyChecking no\n\n" > ~/.ssh/config
+      - run:
+          name: install dockerize
+          command: wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+          environment:
+            DOCKERIZE_VERSION: v0.3.0
+      - run:
+          name: Wait for DB
+          command: dockerize -wait tcp://localhost:3306 -timeout 1m
```
Comment (PatelUtkarsh) [inline .circleci/config.yml]: Reason for adding this?

File: README.md [hunk 2]
```diff
@@ -53,8 +53,24 @@ We can access them in two ways
...
+       "user": "www-data",
+       "path": "/var/www/test.subdomain.com/htdocs"
+     },
+     "develop": {
+       "server": "demo.subdomain.com",
+       "user": "www-data",
+       "path": "/var/www/demo.subdomain.com/htdocs"
+     }
+   }
+   ```
```
Comment (PatelUtkarsh) [inline README.md:76]: Was the test successful with given json?

Comment (vaishaliagola27) [reply README.md:76]: Yes

File: .circleci/config.yml (hunk 1 above)
Comment (vaishaliagola27) [reply .circleci/config.yml]: One pipeline failed because of MySQL was not started yet. So have to add dependency waiting for multiple docker images.

Comment (PatelUtkarsh) [reply .circleci/config.yml]: Can we wait just before installing WP if MySQL docker is not started? Instead of waiting ahead of time?

Comment (PatelUtkarsh) [reply .circleci/config.yml]: Before this line: https://github.com/vaishaliagola27/ci-demo/blob/develop/.circleci/install_test_wp.sh#L20

Comment (vaishaliagola27) [reply .circleci/config.yml]: Done the change. Please check.

**Pull request merged**